src/main/thrift/parquet.thrift

```diff
@@ -958,6 +958,24 @@ union ColumnCryptoMetaData {
 struct ColumnChunk {
   /** File where column data is stored.  If not set, assumed to be same file as
     * metadata.  This path is relative to the current file.
+    *
+    * As of December 2025, the only known use-case for this field is writing summary 
+    * parquet files (i.e. "_metadata" files).  These files consolidate footers from 
+    * multiple parquet files to allow for efficient reading of footers to avoid file 
+    * listing costs and prune out files that do not need to be read based on statistics. 
+    * This is legacy feature as modern table formats (e.g. Iceberg, Hudi and Delta Lake)
+    * are more scalable and serve effectively the same purpose.
+    *
+    * There is no other known use-case for this field. Specifically, there are no known 
```

Comment:
```suggestion
    * There is no other known usage of this field. Specifically, there are no known 
```

I think we should clarify since I believe there are use cases, but there's no implementation.  I don't think it's correct to assert there is no use case.

Reply:
Yep, usage makes sense.